### ISSUE TITLE
Can I use remote-ui instead zoid

### ISSUE THREAD

**Issue by zaharpecherin** [Shopify/remote-ui#67]
I'm developing Shopify app that uses [zoid](https://github.com/krakenjs/zoid) (a cross-domain component toolkit) to render an iframe or popup on shop pages.
On Shopify page added code, that creates zoid component. This component use URL (our server) to retrieve content for the frame.

I'm researching the ability to replace zoid with remote-ui. Could someone help me? Is it possible?

**Comment by lemonmade** [comment]
remote-ui offers some features that are pretty similar to Zoid, but they definitely aren't drop-in replacements for one another. Zoid always maps a component to an iframe, where remote-ui always sandboxes in the component in an invisible container (by default, a web worker). Zoid therefore delegates all rendering to normal HTML/ CSS/ JS that happens in the component's iframe, while remote-ui moves that rendering responsibility to the "host" application. Zoid also comes with conveniences for rendering it directly into an HTML document with no other configuration, while remote-ui requires you to do a bunch of additional runtime code on the host to initialize the code sandbox, and to map the UI from the sandbox to native components.

TL;DR: Zoid and remote-ui are capable of doing some of the same things, but Zoid is probably a lot easier to use if you are OK giving up the rendering control to the component, and if you are OK with the generally-worse performance you will get from having a separate iframe that renders for each UI component.
